src/modules/recipes/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface Recipe {
+  idMeal: string;
+  strMeal: string;
+  strMealThumb: string;
+  strInstructions: string;
+  [key: string]: any; // For other dynamic properties like ingredients
```

Comment:
Make sure to add all other properties and their types that are available in the API response. Then, use that model in all components where the recipe is used, do not declare local Recipe models.